bufferserver/src/main/java/com/datatorrent/bufferserver/internal/DataList.java

```diff
@@ -55,6 +58,9 @@
  */
 public class DataList
 {
+  private static final Logger logger = LoggerFactory.getLogger(DataList.class);
+  private static final ToStringStyle SHORT_CLASS_NAME_STYLE = new ShortClassNameToStringStyle();
```

Comment:
Could you put this style instance in some global place so it can be reused by others?